webapp/www/templates/home.html

```diff
@@ -45,43 +45,43 @@
 <fullpage class="instant-size" page-class="page">
     <section class="page" id="summary">
         <section class="container">
-            <h1>E-Mission</h1>
-            <div class="row">
-            <div class="col-md-2">
-               <img ng-src="img/personal/compare_with_common.png" style="opacity: 0.5" height="480em">
+            <div class="row card" style="padding:1%;">
+            <div class="col-md-2 ">
+               <img ng-src="img/personal/compare_with_common.png" style="opacity: 0.7; float:left;" height="480vw">
             </div>
             <div class="col-md-8" id="main-back">
-            <p class="intro" style="color:brown;">E-Mission is a research project at UC Berkeley that allows all of us to share our travel patterns and experiences with our community and with urban planners.</p>
-            <p class="intro" style="color:brown;">Our goal is to enable personal and structural shifts to sustainable transportation.</p>
-            <p class="intro" style="color:brown;">The project consists of a smartphone application that functions as a personal travel diary, and an aggregate data analysis platform that enables interactive exploration of the collected data.</p>
+            <h1 class="introH">E-Mission</h1>
+            <p class="intro firstPar">E-Mission is a research project at UC Berkeley that allows all of us to share our travel patterns and experiences with our community and with urban planners.</p>
+            <p class="intro firstPar">Our goal is to enable personal and structural shifts to sustainable transportation.</p>
+            <p class="intro firstPar" >The project consists of a smartphone application that functions as a personal travel diary, and an aggregate data analysis platform that enables interactive exploration of the collected data.</p>
             </div>
             <div class="col-md-2">
-               <img ng-src="img/personal/compare_with_common.png" style="opacity: 0.5" height="480em">
+               <img ng-src="img/personal/compare_with_common.png" style="opacity: 0.7; float:right" height="480vw">
             </div>
             </div>
         </section>
     </section>
     <section class="page" id="personal">
         <section class="container">
-          <h1>Personal</h1>
-          <div class="row" style="height: 480em">
+          <div class="row card" style="padding:1%;">
             <div class="col-md-4">
                 <uib-carousel id="personalCarousel" active="active" interval="myInterval" no-wrap="noWrapSlides">
                 <uib-slide ng-repeat="slide in personalSlides track by slide.id" index="slide.id">
                 <img ng-src="{{slide.image}}" style="margin:auto;">
-                <div class="carousel-caption" style="color:brown">
-                  <p style="color:brown">{{slide.text}}</p>
+                <div class="carousel-caption">
```

Comment:
At least for me, the carousel arrows don't show up any more - instead, they show E079. Is this a font issue? See screenshots below.

<img width="381" alt="screen shot 2016-06-13 at 10 56 24 pm" src="https://cloud.githubusercontent.com/assets/2423263/16030768/329a190a-31ba-11e6-9f6d-b2f2b4c3a7fe.png">

<img width="381" alt="screen shot 2016-06-13 at 10 56 57 pm" src="https://cloud.githubusercontent.com/assets/2423263/16030769/32a542c6-31ba-11e6-8141-43b6d43dc9bb.png">